src/main/window-manager.ts

```diff
@@ -1,68 +1,80 @@
-import { reaction } from "mobx";
-import { BrowserWindow, shell } from "electron"
-import windowStateKeeper from "electron-window-state"
 import type { ClusterId } from "../common/cluster-store";
-import { clusterStore } from "../common/cluster-store";
-import { noClustersHost } from "../common/vars";
-import logger from "./logger";
+import { BrowserWindow, ipcMain, shell, WebContents, webContents } from "electron"
+import windowStateKeeper from "electron-window-state"
+import { observable } from "mobx";
+import { initMenu } from "./menu";
 
 export class WindowManager {
-  protected activeView: BrowserWindow;
+  protected mainView: BrowserWindow;
   protected splashWindow: BrowserWindow;
-  protected noClustersWindow: BrowserWindow;
-  protected views = new Map<ClusterId, BrowserWindow>();
-  protected disposers: CallableFunction[] = [];
   protected windowState: windowStateKeeper.State;
 
-  constructor(protected proxyPort: number, showSplash = true) {
+  @observable activeClusterId: ClusterId;
```

Comment:
If adding an observable field, shouldn't the class be marked as `observer`?

Reply:
No, `@observer` is used only with React-components. If component use in `render()` some mobx-observables only then it must have this decorator (to be able to re-render view).

Reply:
Ah thanks for the information, I did not know that